web/src/recent_topics_ui.js

```diff
@@ -17,6 +17,7 @@ import * as message_store from "./message_store";
 import * as message_util from "./message_util";
 import * as message_view_header from "./message_view_header";
 import * as muted_topics_ui from "./muted_topics_ui";
+import * as muted_users from "./muted_users";
 import * as narrow from "./narrow";
```

Comment:
Where is this import used?

Reply:
Yes, it should be in a different commit. I have changed it now. Also I have added test coverage now.